extensions/amp-analytics/0.1/vendors/contentsquare.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS-IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {jsonLiteral} from '../../../../src/json';
+
+const CONTENTSQUARE_CONFIG = jsonLiteral({
+  'vars': {
+    'version': 'amp-0.0.1',
+    'csid':
+      '$MATCH(COOKIE(_cs_id), `([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12})`, 1)',
+    'uuid': '$IF(${csid}, ${csid}, CLIENT_ID(AMP_CID))',
```

Comment:
I wouldn't recommend setting the scope name to `AMP_CID`. This is supposed to be a unique name is scoped to your vendor. `AMP_CID` is too generic and may introduce conflict in the future. 
Some example names are `_ga`, `rakam_device_id``_pressboardmedia`